pkg/sql/subscriber.go

```diff
@@ -186,29 +192,18 @@ func (s *Subscriber) consume(ctx context.Context, topic string, out chan *messag
 
 		case <-time.After(sleepTime): // Wait if needed
 			sleepTime = 0
-
 		}
 
 		messageUUID, noMsg, err := s.query(ctx, topic, out, logger)
-		if err != nil {
-			if isDeadlock(err) {
-				logger.Debug("Deadlock during querying message, trying again", watermill.LogFields{
-					"err":          err.Error(),
-					"message_uuid": messageUUID,
```

Comment:
It seems we will lose the message uuid in logs - an elegant way to keep it would be to add it to the logger using `With` and then pass it down to HandleError.

Reply:
Of course, only if there's any message. :)

Reply:
Yes it's possible indeed.
Though I'm not sure I really get the point of having this UUID (A dead lock is a random sequence of events causing the 2 transactions to get stuck)
But I'll add the with better too much info than not enough :)

Reply:
Yeah, you're right, it probably doesn't make much sense. :) I'm just trying to keep the existing behavior, in case someone would rely on it.